### PR TITLE
CNV-15474: Correcting Incorrect installation doc for disconnected settings

### PR DESCRIPTION
**Pull request by bgaydosrh**
For 4.9, 4.10, and 4.11

Jira: https://issues.redhat.com/browse/CNV-15474
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2034545

Tagging @fabiand for Code Review.
Tagging Debarati Basu Nag for QE Review resource.

Direct doc preview link: https://deploy-preview-43987--osdocs.netlify.app/openshift-enterprise/latest/virt/install/preparing-cluster-for-virt.html

### CODE REVIEW

File: virt/install/preparing-cluster-for-virt.adoc [hunk 1]
```diff
@@ -59,7 +59,7 @@ endif::[]
 
 * To deploy {VirtProductName} in a disconnected environment, you must xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[configure Operator Lifecycle Manager on restricted networks].
 
-* When using a disconnected cluster on a restricted network, you must xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager] to access the Red Hat-provided OperatorHub. Using a proxy allows the cluster to fetch the {VirtProductName} Operator.
+* When using a disconnected cluster on a restricted network, you must xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager] to access the Red Hat-provided OperatorHub. Using a proxy allows the cluster to fetch the {VirtProductName} Operator. A proxy can also be used if the cluster is partially connected to a fully disconnected installation. See xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks] for more information.
```
Comment (fabiand) [inline virt/install/preparing-cluster-for-virt.adoc:62]: I'm not sure what "cluster is partially connected to a fully disconnected installation" means

Comment (fabiand) [reply virt/install/preparing-cluster-for-virt.adoc:62]: I think it should be "A proxy can also be used if the cluster is partially connected."

Comment (bgaydosrh) [reply virt/install/preparing-cluster-for-virt.adoc:62]: Thanks @fabiand - I got that phrase from the Jira record, just to let you know I did not make it up. :-) Changed it to: _A proxy can also be used if the cluster is partially connected._

Also, on the PR you created, yes, a rebase is going to be needed. We made a lot of major attribute changes in the doc set that required all our branches to be rebased recently. That should be completed today.

File: virt/install/preparing-cluster-for-virt.adoc [hunk 2]
```diff
@@ -59,7 +59,7 @@ endif::[]
 
 * To deploy {VirtProductName} in a disconnected environment, you must xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[configure Operator Lifecycle Manager on restricted networks].
 
-* When using a disconnected cluster on a restricted network, you must xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager] to access the Red Hat-provided OperatorHub. Using a proxy allows the cluster to fetch the {VirtProductName} Operator.
+* When using a disconnected cluster on a restricted network, you must xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager] to access the Red Hat-provided OperatorHub. Using a proxy allows the cluster to fetch the {VirtProductName} Operator. A proxy can also be used if the cluster is partially connected. See xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks] for more information.
```
Comment (fabiand) [inline virt/install/preparing-cluster-for-virt.adoc:62]: "When using a partially connected cluster like a cluster in a restricted network, you must"

and drop the whole part "A proxy can also be used if the cluster is partially connected. See xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks] for more information." as this is pretty redundant to line 60.

Comment (bgaydosrh) [reply virt/install/preparing-cluster-for-virt.adoc]: Per our GChat we agreed to:

With limited internet connectivity, you can xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager] to access the Red Hat-provided OperatorHub. If you are using a restricted network with no internet connectivity, you must xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Configure Operator Lifecycle Manager for restricted networks].

If this looks good, @fabiand please re-approve. Thanks!

**Comment by dbasunag** [comment]
lgtm

File: virt/install/preparing-cluster-for-virt.adoc [hunk 3]
```diff
@@ -59,7 +59,7 @@ endif::[]
 
 * To deploy {VirtProductName} in a disconnected environment, you must xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[configure Operator Lifecycle Manager on restricted networks].
 
-* When using a disconnected cluster on a restricted network, you must xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager] to access the Red Hat-provided OperatorHub. Using a proxy allows the cluster to fetch the {VirtProductName} Operator.
+* With limited internet connectivity, you can xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager] to access the Red Hat-provided OperatorHub. If you are using a restricted network with no internet connectivity, you must xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Configure Operator Lifecycle Manager for restricted networks].
```
Comment (sjhala-ccs) [inline virt/install/preparing-cluster-for-virt.adoc:62]: Isn't the second sentence the same as the previous bullet on line 60?

Comment (sjhala-ccs) [inline virt/install/preparing-cluster-for-virt.adoc:62]: Suggestion:

s/With limited internet connectivity/If you have limited internet connectivity

Comment (sjhala-ccs) [inline virt/install/preparing-cluster-for-virt.adoc:62]: s/Configure/configure in the link text for the second xref

Comment (bgaydosrh) [reply virt/install/preparing-cluster-for-virt.adoc:62]: Yep, we can get rid of line 60. The new wording is a much more descriptive summary.

Comment (bgaydosrh) [reply virt/install/preparing-cluster-for-virt.adoc:62]: Done.

Comment (bgaydosrh) [reply virt/install/preparing-cluster-for-virt.adoc:62]: Done.

**Pull request merged**